zebra/zebra_opaque.c

```diff
@@ -613,13 +717,185 @@ static int handle_opq_unregistration(const struct zmsghdr *hdr,
 	return ret;
 }
 
+/*
+ * Handle requests about opaque notifications.
+ */
+static int handle_opq_notif_req(const struct zmsghdr *hdr, struct stream *msg)
+{
+	int ret;
+	struct zapi_opaque_notif_info info = {};
+	struct opq_client_reg *client;
+	struct opq_msg_reg key, *reg;
+	char buf[50];
```

Comment:
yeah, I guess I was wanting to avoid the work of zero-ing that buffer. the `2str` function does an unconditional snprintf into the buffer, so  I don't think there's a way we could have junk?